src/js/components/TeamDetail.js

```diff
@@ -0,0 +1,230 @@
+import Api from "../api/api.js";
+import {$, $$} from "../utils/querySelector.js";
+
+export default function teamDetail() {
+  this.teamId = "";
+
+  const setMemberList = async () => {
+    addMembers(await getTeamMembers());
+  }
+
+  const getTeamMembers = async () => {
+    const params = new URLSearchParams(window.location.search);
+    this.teamId = params.get("id");
+    const data = await Api.getFetch(`/api/teams/${this.teamId}`);
+
+    $("#user-title").querySelector("strong").innerHTML = data.name;
+
+    return data.members;
+  };
+
+  const addEvent = () => {
+    const $listContainer = $(".todoapp-list-container");
+
+    $listContainer.addEventListener('click', ({target, target: {value, classList}}) => {
+
+      if (classList.contains('destroy')) deleteItem(target)
+      else if (classList.contains('toggle')) completeItem(target)
+      else if (classList.contains('clear-completed')) deleteAll(target)
+      else if (classList.contains('filter')) filterList(target)
+    });
+
+    $listContainer.addEventListener('change', ({target, target: {value, classList}}) => {
+      if (classList.contains('chip')) changePriority(target, value);
+    });
+
+    $listContainer.addEventListener('dblclick', ({target, target: {value, classList}}) => {
+      if (classList.contains('label')) editItem(target);
+    });
+
+    $listContainer.addEventListener('keyup', ({target, target: {value, classList}, key}) => {
+      key === "Enter" && addTodo(target);
+    });
+
+    $('#add-user-button').addEventListener('click', async () => {
+      const result = prompt('새로운 팀원 이름을 입력해주세요');
+
+      await Api.postFetch(`/api/teams/${this.teamId}/members`, {name: result});
+      await setMemberList();
+    });
+  };
+
+  const addTodo = async ($newTodo) => {
+    const content = $newTodo.value;
+    const memberId = $newTodo.closest(".todoapp-container").dataset.memberId;
+
+    if (content.length < 2) return alert(Message.SHORT_INPUT_LENGTH);
+
+    await Api.postFetch(`/api/teams/${this.teamId}/members/${memberId}/items/`, {contents: content});
```

Comment:
value의 이름이 contents 였다면 
```suggestion
    await Api.postFetch(`/api/teams/${this.teamId}/members/${memberId}/items/`, { contents });
```
위의 코드처럼 줄일 수 있었겠네요!!